src/libraries/Microsoft.Extensions.Configuration.Abstractions/src/ConfigurationExtensions.cs

```diff
@@ -81,5 +81,32 @@ public static bool Exists(this IConfigurationSection section)
             }
             return section.Value != null || section.GetChildren().Any();
         }
+
+
+        /// <summary>
```

Comment:
(nit) no need for a double blank line above this. Just need one blank line in between methods.

Reply:
done